hosting_docs/source/reference/howto/wipe_persistent_data.rst

```diff
@@ -69,42 +74,25 @@ Rebuilding environment
 ----------------------
 
 
-#. 
-   Drop the first line public.yml. This step assumes the first line is
-   "wipe_environment_enabled: True", prepended in the steps above.
-
-   .. code-block::
-
-      $ tail -n +2 ~/environments/monolith/public.yml > public.yml.tmp
-      $ mv public.yml.tmp ~/environments/monolith/public.yml
+#. Remove the "wipe_environment_enabled: True" line in your `public.yml` file.
 
-#. 
-   Run Ansible playbook to recreate databases
+#. Run Ansible playbook to recreate databases
 
    .. code-block::
 
-      $ cchq monolith ap deploy_db.yml
+      $ cchq <env_name> ap deploy_db.yml
```

Comment:
Has this worked for you? In the past I have had to add the `--skip-check` flag (if I understand correctly, the `--skip-check` flag specifies that the command is not a dry run).